test/ext/base_lint.py

```diff
@@ -169,6 +169,21 @@ def test_other_stuff():
                         pass
                 ''')
 
+            class CaseChainedAssign(IncludesUnexpectedFocused):
+                # language=py
+                source = lambda_fixture(lambda pytestmark_decl: f'''
+                    import pytest
+
+                    pytestmark = stuff = {pytestmark_decl}
+
+                    class TestMyStuff:
+                        def test_stuff(self):
+                            pass
+
+                    def test_other_stuff():
+                        pass
+                ''')
+
```

Comment:
Since each chained assignment target would get the same value, I would alter the test here to check for `pytestmark = x = ...` _and_ `x = pytestmark = ...`

```suggestion
            class CaseChainedAssign(IncludesUnexpectedFocused):
                assignments = lambda_fixture(params=[
                    pytest.param(('pytestmark', 'stuff'), id='pytestmark-first'),
                    pytest.param(('stuff', 'pytestmark'), id='pytestmark-last'),
                ])

                # language=py
                source = lambda_fixture(lambda pytestmark_decl, assignments: f'''
                    import pytest

                    {" = ".join(assignments)} = {pytestmark_decl}

                    class TestMyStuff:
                        def test_stuff(self):
                            pass

                    def test_other_stuff():
                        pass
                ''')

```